recipe/tests/cmake/CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
 # https://martinopilia.com/posts/2018/09/15/building-python-extension.html
+# FindPythonInterp and FindPythonLibs is deprecated since cmake 3.12
```

Comment:
That doesn't mean it's not used by projects. Please keep the test as it is and add another if you want to.

Reply:
Sure, I'll add them back.

Reply:
@isuruf could you maybe have a look at the CI logs and help me understand why the builds are failing with `FindPythonInterp` and `FindPythonLibs`? That's why I switched to `FindPython` in the first place.

Reply:
Resolved in future commit